Cython/Compiler/Parsing.py

```diff
@@ -3077,18 +3087,50 @@ def p_with_gil(s):
     else:
         return 0
 
-def p_exception_value_clause(s):
+def p_exception_value_clause(s, ctx):
+    """
+    Parse exception value clause.
+
+    Maps clauses to exc_check / exc_value / exc_clause as follows:
```

Comment:
`exc_clause` is just for the `nogil` order warning I think? That's fine, but I think it'd be good to mention that in the comment. Just because `exc_check` and `exc_value` are much more important to how Cython handles exceptions and I think it's worth making that clear

Reply:
While it's true that _currently_ `exc_clause` is only used to correctly raise the `nogil` warning, it could _potentially_ be useful for other things in the future? I just feel it's very easy for the comment here to become out of sync with how the return value is actually used outside this function.

Reply:
The comment contains a table with a rather complete specification. If anything at all changes in the implementation, it will most likely need to be adapted. I don't think more information will make that any worse.

Reply:
Thanks! Added a note.